.changeset/twelve-moles-dig.md

```diff
@@ -0,0 +1,10 @@
+---
+"@commercetools/mcp-essentials": major
+"@commercetools/agent-essentials": major
+---
+
+Add support for authencation via existing AuthToken.
```

Comment:
Nit: we can add a migration example here with before and after maybe even using 

```diff
- Prior
+ After
```

When updating people often navigate to GitHub releases and concrete easy steps help. Totally optional though!